### PR TITLE
Removes under barrel stuff from SH-15

### PR DESCRIPTION
**Pull request by Arctrooper8**
## About The Pull Request

Removes the plasma pistol from the SH-15

## Why It's Good For The Game

The SH-15 has a unique niche where it has access to under barrel attachments, it makes up for this with it's specialized ammo types which comes in mags and do less damage (alongside not having buckshot at all).
However the plasma pistol is arguably still too much even for a the more reserved magazine fed cousin of the other shotguns.
What with it's on hit bonus damage and fire stacks, the plasma pistol combined with any low fire rate high damage per shot weapon is a strong and generally unfun to play against combination. (One is left to wonder why it even became an attachment in the first place)

TLDR: SH-15 having under barrels good niche, plasma pistol bad and dumb and stupid and too strong why is it even an attachment.

## Changelog
:cl:
balance: Removed plasma pistol as an attachment option for the SG-15
/:cl:

### CODE REVIEW

**Comment by Naaanii** [comment]
Pretty sure TX-15 could always have underbarrel stuff before more powercreep was introduced. It's also the only shotgun that takes magazines. I think having underbarrel is a good niche for this gun.

**Comment by Lumipharon** [comment]
Only the plasma pistol is an issue, if at all. The rest are entirely non issues.

It was a deliberate choice to keep underbarrels on it.

It uses different (weaker) ammo that normal shotguns, the slugs have notably worse CC. It's coded as a rifle as that's functionally what it is.

**Comment by Arctrooper8** [comment]
> Only the plasma pistol is an issue, if at all. The rest are entirely non issues.
> 
> It was a deliberate choice to keep underbarrels on it.
> 
> It uses different (weaker) ammo that normal shotguns, the slugs have notably worse CC. It's coded as a rifle as that's functionally what it is.

I think both fire attachments are an issue, as while the SG-15 slug may not be as strong as the normal SH slug it still causes knockback which stuns you and makes you much easier to hit with the plasma pistol and flamer.
And the UGL niche is already taken by the italian shotgun (which is seasonal to be fair)
And let's be honest, nobody uses the mini pepperball gun.

**Comment by Naaanii** [comment]
> > Only the plasma pistol is an issue, if at all. The rest are entirely non issues.
> > It was a deliberate choice to keep underbarrels on it.
> > It uses different (weaker) ammo that normal shotguns, the slugs have notably worse CC. It's coded as a rifle as that's functionally what it is.
> 
> I think both fire attachments are an issue, as while the SG-15 slug may not be as strong as the normal SH slug it still causes knockback which stuns you and makes you much easier to hit with the plasma pistol and flamer. And the UGL niche is already taken by the italian shotgun (which is seasonal to be fair) And let's be honest, nobody uses the mini pepperball gun.

It knocks back, but doesn't stun, slowdown is a given as the weapon relies on landing consistent shots. Once again this weapon is more like a rifle rather than an actual shotgun so having underbarrel attachments on it is fine. If anything, deleting plasma pistol would be the only solution if it was an actual issue as the auto shottie has had ubf for a long time and while strong, it has never been an actual issue.

**Comment by lovegreenstuff** [comment]
hey what's up guys it's ya boi

uhhh the original shotgun underbarrel pr that i made for pette intentionally left out tx15 because it was mostly a matter of burst damage, aka buck, being combo'd into shit like another buck or an impact nade and basically oneshotting people

tx15 only does like 85 damage max per shot compared to like 170 for t39 buck, so we just thought it wasnt terribly necessary

since then, impact nades got sent to the shadow realm, flamer damage on hit was set to zero, and masterkey firerate got nerfed, so it's probably less necessary to do so now.

but hey if you want to fuckin uhhhhh keep goin' with this pr, go for it mang

**Comment by nutsupreme** [comment]
salt pr

**Review by Pariah919 — CHANGES_REQUESTED** [review]
50/50 on Plasma pistol, I'm fine with it being gone, but I feel T18 carbine is pertty equivalent anyway.
Hard no on the rest of them.

**Comment by Arctrooper8** [comment]
The people have spoken. Considering that the SH-15 has different ammo which does reduced damage, and the unique niche having under barrels gives this gun, it is obviously better to let them keep the under barrel attachments.
Except the plasma pistol, because it sucks.
That repulsive duct taped DIY puke green fire pistol, it shouldn't have ever been an attachment.

**Comment by BraveMole** [comment]
SH 15 needs it niche, and having attachments is fine
